docs/source/quickstart.mdx

```diff
@@ -0,0 +1,389 @@
+
+As a best practice, your gateway should _not_ be responsible for composing its own supergraph schema. Instead, a separate process should compose the schema and provide it to the gateway. This helps improve reliability and reduce downtime when you make changes to a subgraph.
+
+There are multiple ways to compose a supergraph schema from our subgraph schemas:
+
+* On our local machine using the Rover CLI (we'll start with this)
+* Via managed federation in Apollo Studio (we'll switch to this in a later section)
+
+Using managed federation is strongly recommended for production environments. We'll start with local composition just to get up and running.
+
+### Obtaining subgraph details
+
+To compose our supergraph schema, we need the following information about each of our subgraphs:
+
+* The subgraph's schema
+* The URL of the subgraph's GraphQL endpoint (which must be accessible by the gateway)
+
+We'll start with just two subgraphs. Because we're using Apollo-hosted example services for our subgraphs, we know their endpoint URLs. We _don't_ have their schemas, however.
+
+#### Fetching subgraph schemas
+
+We can fetch a running subgraph's schema by executing an enhanced introspection query on it. The Rover CLI provides a command to do exactly that.
+
+Run the following command in your terminal:
+
+```shell
+rover subgraph introspect https://7bssbnldib.execute-api.us-east-1.amazonaws.com/Prod/graphql
+```
+
+Rover introspects Apollo's example `products` service and outputs the following schema:
+
+<ExpansionPanel title="Click to expand">
+
+```graphql
+enum CURRENCY_CODE {
+  USD
+}
+
+type Department {
+  category: ProductCategory
+  url: String
+}
+
+type Money {
+  amount: Float
+  currencyCode: CURRENCY_CODE
+}
+
+"""Here are some helpful details about your type"""
+type Price {
+  cost: Money
+
+  """A number between 0 and 1 signifying the % discount"""
+  deal: Float
+  dealSavings: Money
+}
+
+"""
+This is an Entity, docs:https://www.apollographql.com/docs/federation/entities/
+You will need to define a __resolveReference resolver for the type you define, docs: https://www.apollographql.com/docs/federation/entities/#resolving
+"""
+type Product @key(fields: "id") {
+  id: ID!
+  title: String
+  url: String
+  description: String
+  price: Price
+  salesRank(category: ProductCategory = ALL): Int
+  salesRankOverall: Int
+  salesRankInCategory: Int
+  category: ProductCategory
+  images(size: Int = 1000): [String]
+  primaryImage(size: Int = 1000): String
+}
+
+enum ProductCategory {
+  ALL
+  GIFT_CARDS
+  ELECTRONICS
+  CAMERA_N_PHOTO
+  VIDEO_GAMES
+  BOOKS
+  CLOTHING
+}
+
+extend type Query {
+  bestSellers(category: ProductCategory = ALL): [Product]
+  categories: [Department]
+  product(id: ID!): Product
+}
+```
+
+</ExpansionPanel>
+
+Now from your project directory, run that same command, but this time append ` > products.graphql`. This writes the subgraph's schema to a file instead of printing it to your terminal.
```

Comment:
maybe include the full command after this so folks don't need to scroll back up?